rust/agama-software/src/lib.rs

```diff
@@ -41,18 +41,14 @@ pub use start::start;
 pub mod service;
 pub use service::Service;
 
-mod model;
+pub mod model;
```

Comment:
why model needs to be accessible outside of crate?

Reply:
You are right, it is not. I will fix it.